src/nupic/ntypes/Collection.hpp

```diff
@@ -63,6 +64,7 @@ class Collection {
     return !operator==(other);
   }
   inline size_t getCount() const { return vec_.size(); }
+  inline size_t size() const { return vec_.size(); }
```

Comment:
rm either size(), or getCount() as those are the same.

Reply:
> rm either size(), or getCount() as those are the same.

size( ) is a start toward migration away from Collection.  getCount( ) is to keep API compatibility.